frontend/src/lib/components/AuthorizedUrlList/authorizedUrlListLogic.ts

```diff
@@ -37,6 +37,16 @@ export enum AuthorizedUrlListType {
     RECORDING_DOMAINS = 'RECORDING_DOMAINS',
 }
 
+/**
+ * Firefox does not allow you construct a new URL with e.g. https://*.example.com (which is to be fair more standards compliant than Chrome)
+ * when used to probe for e.g. for authorized urls we only care if the proposed URL has a path so we can safely replace the wildcard with a character
+ * NB this changes its input and shouldn't be used for general purpose URL parsing
+ */
+export function sanitizePossibleWildCardedURL(url: string): URL {
```

Comment:
brought all the wildcard specific stuff into `authorizedUrlsListLogic` since we shouldn't care about wildcards generally in util functions